tasks/task_122_winowhy_reason_plausibility_detection.json

```diff
@@ -0,0 +1,2892 @@
+
+{
+    "Contributors": [
+        "Xinran Zhao",
+        "Hongming Zhang",
+        "Yangqiu Song"
+    ],
+    "Source": [
+        "winowhy"
+    ],
+    "Categories": [
+        "Classification"
+    ],
+    "Definition": "In this task you need to indicate the plausibility of reasoning for the pronoun coreference relations. The given sentences are provided by a sentence with a target pronoun and a sentence that justifies which noun phrase the pronoun refers to. The resolution of the pronoun coreference relations typically involve one or multiple following knowledge types about commonsense: First: 'Property', the knowledge about property of objects (e.g., ice is cold). Second: 'Object', the knowledge about objects (e.g., cats have ears). Third: 'Eventuality', the knowledge about eventuality (e.g., 'wake up' happens before 'open eyes'). Forth: 'Spatial', the knowledge about spatial position (e.g., object at the back can be blocked). Fifth: 'Quantity', the knowledge about numbers (e.g., 2 is smaller than 10). Sixth: all other knowledge if above ones are not suitable. Indicate with `Correct` if the reasoning is correct. Indicate with `Wrong`, otherwise. ",
```

Comment:
This definition feels like overkill for this task. In particular, I am not sure if you need the definitions for `Property`, `Eventuality`, etc to define and answer this task.

Reply:
Thank you very much for the comments. I still followed the template for task 010 where more information is given, yet I can also remove these definitions for the categories.

Reply:
My suggestion is to include the details needed for this task. If the category definitions are not needed (say, for an average human), I think we should drop them.

Reply:
Sure, I deleted it for the tasks that might not require the definition and add some explanation to smooth the transition of the category definitions.